addons/mail/static/tests/chatter_tests.js

```diff
@@ -1248,6 +1248,74 @@ QUnit.test('form activity widget: schedule next activity', function (assert) {
     form.destroy();
 });
 
+
+QUnit.test('form activity widget: edit next activity', function (assert) {
+    assert.expect(3);
+    var self = this;
+    this.data.partner.records[0].activity_ids = [1];
+    this.data.partner.records[0].activity_state = 'today';
+    this.data['mail.activity'].records = [{
+        id: 1,
+        display_name: "An activity",
+        date_deadline: moment().format("YYYY-MM-DD"), // now
+        state: "today",
+        user_id: 2,
+        create_user_id: 2,
+        activity_type_id: 2,
+    }];
+
+    var form = createView({
+        View: FormView,
+        model: 'partner',
+        data: this.data,
+        services: this.services,
+        arch: '<form string="Partners">' +
+                '<sheet>' +
+                    '<field name="foo"/>' +
+                '</sheet>' +
+                '<div class="oe_chatter">' +
+                    '<field name="message_ids" widget="mail_thread"/>' +
+                    '<field name="activity_ids" widget="mail_activity"/>' +
+                '</div>' +
+            '</form>',
+        res_id: 2,
+        mockRPC: function (route, args) {
```

Comment:
no need to specify a mockRPC if it only calls _super

Reply:
forgot this one thanks. fixed